src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -342,4 +342,19 @@ public void dropDatabase(String databaseName) {
         MongoDatabase db = mongoClient.getDatabase(databaseName);
         db.drop();
     }
+
+    /**
+     * Returns a boolean indicating that if MongoDB is up and running or not.
```

Comment:
You can remove "that" from the sentence.

Reply:
Done.